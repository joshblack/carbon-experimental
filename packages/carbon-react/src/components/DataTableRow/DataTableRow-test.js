import React from 'react';
import DataTableRow from '../DataTableRow';
import { shallow } from 'enzyme';

describe('DataTableRow', () => {
  it('should render', () => {
    const wrapper = shallow(
      <DataTableRow>
        <div>Children</div>
      </DataTableRow>
    );
    expect(wrapper).toMatchSnapshot();
  });

  it('should render children as expected', () => {
    const wrapper = shallow(
      <DataTableRow>
        <div>Children</div>
      </DataTableRow>
    );
    expect(wrapper.find('div').length).toEqual(1);
  });
});
