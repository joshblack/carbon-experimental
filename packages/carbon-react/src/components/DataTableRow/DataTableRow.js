import PropTypes from 'prop-types';
import React from 'react';

const DataTableRow = props => {
  const { className, children, ...other } = props;

  return (
    <tr {...other} className={className}>
      {children}
    </tr>
  );
};

DataTableRow.propTypes = {
  children: PropTypes.node,
  className: PropTypes.string,
};

export default DataTableRow;
