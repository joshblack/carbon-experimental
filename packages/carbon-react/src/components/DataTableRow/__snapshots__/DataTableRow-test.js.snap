// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataTableRow should render 1`] = `
<tr>
  <div>
    Children
  </div>
</tr>
`;
